app/components/avo/panel_component.html.erb

```diff
@@ -9,6 +9,11 @@
         <% end %>
         <div class="text-2xl tracking-normal font-semibold text-gray-800 truncate items-center flex flex-1" data-target="title">
           <span><%= @name %></span>
+          <% if @reload_button %>
+            <%= button_tag data: { controller: "panel-refresh", action: "click->panel-refresh#refresh" }, id: "reloadButton" do %>
```

Comment:
Let's try to not add `id`s to elements unless we really need them.

We can still query for this button (find in page) with the following selector `button[data-controller="panel-refresh"]`.

```suggestion
            <%= button_tag data: { controller: "panel-refresh", action: "click->panel-refresh#refresh" } do %>
```

Reply:
Ok, I will update that.

Reply:
This has not been addressed